planner/core/logical_plan_builder.go

```diff
@@ -4243,10 +4258,9 @@ func (b *PlanBuilder) tryBuildCTE(ctx context.Context, tn *ast.TableName, asName
 			lp.SetSchema(getResultCTESchema(cte.seedLP.Schema(), b.ctx.GetSessionVars()))
 
 			if cte.recurLP != nil && cte.isInline {
-				b.ctx.GetSessionVars().StmtCtx.AppendWarning(ErrInternal.GenWithStack("Hint merge() is inapplicable. Please check whether the CTE use recursive."))
+				b.ctx.GetSessionVars().StmtCtx.AppendWarning(ErrInternal.GenWithStack("Recursive cte cannot be inlined."))
```

Comment:
cannot -> can not

Reply:
cte -> CTE

Reply:
Fixed. BTW, what's different between 'cannot' and 'can not' ?

Reply:
Ah, no difference. I just realized `cannot` is also acceptable.